henge.go

```diff
@@ -28,11 +29,29 @@ func convertToVersion(objs []runtime.Object, version string) ([]runtime.Object,
 	return ret, nil
 }
 
+// Loop over a array of filepaths and check if it exists
+// if it exists check if it is not a directory.
+func ifFileExists(files []string) {
+	for _, filename := range files {
+		fileInfo, err := os.Stat(filename)
+		if err != nil {
+			glog.Fatalf("File %s not found.", filename)
```

Comment:
This prints also time and filename with line number, I don't think that user needs that.
You can use `fmt.Fprintf(os.Stderr, ....)` to print to stderr.